puppet/modules/slave/manifests/postgresql.pp

```diff
@@ -1,22 +1,38 @@
 # @api private
 class slave::postgresql {
-  Postgresql_psql {
-    cwd => '/',
-  }
+  # only CentOS slaves are used to run unit tests
+  if $::osfamily == 'RedHat' {
+    if $::operatingsystemmajrelease == '7' {
 
-  include ::postgresql::client
-  include ::postgresql::server
+			exec { 'remove postgresql':
+				command => "/bin/yum -y remove postgresql-server postgresql-client postgresql",
```

Comment:
It should be possible to use the `Package` resource for these since the postgresql module now defines the SCL packages, right?

Reply:
I tried that, got duplicate resource definition errors.